sdk/python/feast/feature_store.py

```diff
@@ -142,8 +141,9 @@ def __init__(
         if registry_config.registry_type == "sql":
             self._registry = SqlRegistry(registry_config, None)
         else:
-            self._registry = Registry(registry_config, repo_path=self.repo_path)
-            self._registry._initialize_registry()
+            r = Registry(registry_config, repo_path=self.repo_path)
```

Comment:
why is this change necessary?

Reply:
`self._registry` has type `BaseRegistry` which doesn't have an `_initialize_registry` method.

Reply:
`self._registry` has type `BaseRegistry` which doesn't have an `_initialize_registry` method.